bind_test.go

```
@@ -1719,9 +1756,17 @@ func Test_Bind_RepeatParserWithSameStruct(t *testing.T) {
 		require.NoError(t, c.Bind().Body(r))
 		require.Equal(t, "body_param", r.BodyParam)
 	}
+	cb, err := cbor.Marshal(&Request{
+		BodyParam: "body_param",
+	})
+
```

golangci-lint (1.62.2)

File is not `gofumpt`-ed with `-extra`

(gofumpt)